pom.xml

```diff
@@ -46,25 +50,14 @@
     </modules>
 
     <properties>
-        <jersey.version>2.19</jersey.version>
-        <confluent-common.version>3.3.0-SNAPSHOT</confluent-common.version>
-        <kafka.version>0.11.0.0-SNAPSHOT</kafka.version>
-        <kafka.scala.version>2.11</kafka.scala.version>
-        <log4j.version>1.7.6</log4j.version>
-        <restutils.version>3.3.0-SNAPSHOT</restutils.version>
+        <apache.directory.server.version>2.0.0-M22</apache.directory.server.version>
+        <apache.directory.api.version>1.0.0-M33</apache.directory.api.version>
         <avro.version>1.7.7</avro.version>
+        <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
+        <exec-maven-plugin.version>1.2.1</exec-maven-plugin.version>
         <jackson.version>2.5.4</jackson.version>
```

Comment:
my thought was move everything as high as necessary. so this should be retrieved from rest-utils.

in fact this was just an oversight. we don't need/use this in schema-registry anymore